lib/init.g

```diff
@@ -1018,8 +1030,11 @@ end );
 
 if IsHPCGAP and THREAD_UI() then
   ReadLib("hpc/consoleui.g");
-  MULTI_SESSION();
-else
+  if IsLIBGAP = false then
+    MULTI_SESSION();
+  fi;
+elif IsLIBGAP = false then
+  BIND_CONSTANT( "IsLIBGAP", false );
```

Comment:
Actually, this call to `BIND_CONSTANT` is redundant, isn't it? After all, you just checked that value.

Reply:
It was supposed to make `IsLIBGAP` constant, which does not work in this case anyway, so I will remove it.